src/controllers/reactWebviewViewController.ts

```diff
@@ -76,9 +76,7 @@ export class ReactWebviewViewController<State, Reducers>
 
             localResourceRoots: [this._context.extensionUri],
         };
-        this._webviewView.onDidDispose(() => {
-            this.dispose();
```

Comment:
Is `onDidDispose` also fired during webviewview drag?
![image](https://github.com/user-attachments/assets/d4464206-d21e-416b-bf64-fbf279c8ed5b)

Reply:
Yes, it gets fired during a drag event

Reply:
Wow that's unexpected but I guess that's just how vscode implemented this, the fix LGTM and I think not being able to clean up resource for the use case I underlined in the screenshot is acceptable.